components/scg/src/scg-render.js

```diff
@@ -39,10 +58,14 @@ SCg.Render.prototype = {
                 self.scene.onMouseUpObject(d);
                 if (d3.event.stopPropagation()) d3.event.stopPropagation();
             })
-            .on("wheel", function(){
+            .on("wheel", function () {
                 var direction = d3.event.wheelDelta < 0 ? 'down' : 'up';
-                if (direction === 'up') self.scene.onMouseWheelUp(this, self);
-                if (direction === 'down') self.scene.onMouseWheelDown(this, self);
+                if (direction === 'up') {
+                    self.transformByZoom(d3.event);
+                };
```

Comment:
```suggestion
                }
```
Please fix here and later